<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordPage</class>
 <widget class="QDialog" name="ChangePasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="changePasswordPageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>260</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: blue; color: white;</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentPasswordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>Current Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="newPasswordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>New Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmNewPasswordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>Confirm New Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter current password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter new password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmNewPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm new password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePasswordButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: blue;</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background: lightgray;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="successLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green;</string>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 